tests/Backend/CommonPart1/CreateTableTest.php

```diff
@@ -250,6 +250,20 @@ public function testTableWithUnsupportedCharactersInNameShouldNotBeCreated(): vo
         }
     }
 
+    public function testBucketWithUnsupportedCharactersInNameShouldNotBeValid(): void
+    {
+        try {
+            $this->_client->createTable(
```

Comment:
nemohl by to byt `createTableAsync` ? at prestaneme pouzivat deprecovane metody

Reply:
Done.